src/pages/promises/index.js

```diff
@@ -112,12 +113,20 @@ export async function getStaticProps({ locale }) {
 
   const page = await wp().pages({ slug: "promises", locale }).first;
   const { promiseStatuses } = page;
-  const checkApi = check({
-    promiseStatuses,
-    team: "pesacheck-promise-tracker",
-  });
-  const projectMeta = await checkApi.projectMeta();
-  const promises = await checkApi.promises({
+  const promiseSourceLibs = {
+    Json: JsonpromiseSource(),
+    Check: check({
+      promiseStatuses,
+      team: "pesacheck-promise-tracker",
+    }),
+  };
+
+  const sourceLib = process.env.SOURCE_LIB || "Check";
+
+  const api = promiseSourceLibs[sourceLib];
```

Comment:
Rather than doing this on every page, can't the  `promisesSourceLibs` (probably renamed to `promisesApi`) check the `env` on initialisation? i.e. one only need to call `promisesApi()`.

Reply:
@kilemensi  as I continue working on this, I think the functionality is ready for re-review.

Reply:
☝🏾 still not done @esirK

Reply:
this has now been done.